superset-frontend/src/explore/components/ControlHeader.tsx

```diff
@@ -40,6 +40,16 @@ export type ControlHeaderProps = {
   danger?: string;
 };
 
+const iconStyles = css`
```

Comment:
This is fine for now, but let's talk about this chunk in person sometime. I'd love to be in a state where we don't need to be adding/overriding/tweaking Icon styles in any particular use/implementation, instead adding classes/config/props on the Icon component itself to handle these needs more globally.